core/src/main/scala/org/http4s/headers/Accept.scala

```diff
@@ -1,12 +1,25 @@
 package org.http4s
 package headers
 
+import org.http4s.util.{Renderable, Writer}
+
 import scalaz.NonEmptyList
 
-object Accept extends HeaderKey.Internal[Accept] with HeaderKey.Recurring
+object Accept extends HeaderKey.Internal[Accept] with HeaderKey.Recurring {
+  implicit def withDefaultQValue(mediaRange: MediaRange): MediaRangeAndQValue =
```

Comment:
Ya, I gather you are not a fan?
On Feb 21, 2016 4:17 PM, "Bryce Anderson" <notifications@github.com> wrote:

> In core/src/main/scala/org/http4s/headers/Accept.scala
> <https://github.com/http4s/http4s/pull/532#discussion_r53576111>:
>
> >  import scalaz.NonEmptyList
> >
> > -object Accept extends HeaderKey.Internal[Accept] with HeaderKey.Recurring
> > +object Accept extends HeaderKey.Internal[Accept] with HeaderKey.Recurring {
> > +  implicit def withDefaultQValue(mediaRange: MediaRange): MediaRangeAndQValue =
>
> Whats the point of the implicit here? for implicit conversion?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/http4s/http4s/pull/532/files#r53576111>.
>